model/ftn/ww3_grid.ftn

```diff
@@ -2582,6 +2585,15 @@
 !/MLIM      J      = 1
       WRITE (NDSO,967) FHMAX, FHMAX/SQRT(2.), YESXNO(J)
       IF ( FHMAX.LT.0.50 .AND. J.EQ.1 ) WRITE (NDST,968)
+!
+      IF ( CAL360 ) THEN
```

Comment:
Might it be better to deploy the catch for when CAL360 and NOLEAP are set true, used at line 6293 here? As written I think the code would only deal with this problem if reading from the .nml file?

Reply:
Thanks Juan, the new changes using CALTYPE look good here